requirements.txt

```diff
@@ -0,0 +1,337 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Is this handling the extras too? It should account for `[telemetry,test]`

Reply:
I'm okay adding telemetry, but the function of the tool does not depend on the versions of test dependencies, so I'd rather not set hard pins.